src/lib.rs

```diff
@@ -24,3 +26,5 @@ pub use egui_suspense as suspense;
 pub use egui_thumbhash as thumbhash;
 #[cfg(feature = "virtual_list")]
 pub use egui_virtual_list as virtual_list;
+#[cfg(feature = "material_icons_proc_macro")]
+pub use material_icons_proc_macro;
```

Comment:
I don't think anyone would want to use the proc macro directly, so I don't think we need to expose it here.